cfgov/unprocessed/js/modules/util/dom-manipulators.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var domTraverse = require( './dom-traverse' );
+
+/**
+ * Shortcut for creating new dom elements
+ * @param   {string} tag     The html elem to create
+ * @param   {object} options The options for building the elem
+ * @returns {HTMLNode}       The created elem
+ */
+function create( tag, options ) {
+  var elem = document.createElement( tag );
```

Comment:
Could you clarify? To me, traversal is completely different from manipulation and should live on it's own. Also, by requiring the other files within traverse we end up including all the manipulation files anyway, which doesn't seem to save us anything.